utils/_context/_scenarios/__init__.py

```diff
@@ -783,7 +783,12 @@ class _Scenarios:
 
     runtime_metrics_enabled = EndToEndScenario(
         "RUNTIME_METRICS_ENABLED",
+        weblog_env={"DD_DOGSTATSD_START_DELAY": "0"},
```

Comment:
Could you add a comment explaining that this is necessary to avoid the default startup delay on Java, which is there to reduce the impact on startup and avoid side-effects in various application servers (using JMX will trigger loading of various JDK features like logging which can interfere with custom log managers set during startup)

Reply:
Done!